Makefile

```diff
@@ -65,11 +65,13 @@ testbed-correctness: otelcol
 
 .PHONY: testbed-list-loadtest
 testbed-list-loadtest:
-	TESTBED_CONFIG=local.yaml $(GOTEST) -v ./testbed/tests --test.list '.*'|head -n 20
+    # Use this hack to reverse, tail then reverse again because `head -n -1` does not work on macos.
+	TESTBED_CONFIG=local.yaml $(GOTEST) -v ./testbed/tests --test.list '.*'| tail -r | tail -n +2 | tail -r
```

Comment:
why don't use grep instead of counting lines? as done here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/master/testbed/Makefile#L5

Reply:
Not counting the lines, but I think grep may be better, let me give a try

Reply:
Done, works good.